controllers/recipes.js

```diff
@@ -155,8 +155,17 @@ module.exports = {
   },
   searchRecipe: async (req, res) => {
     console.log("🔎 The search button is working.");
+    console.log("Query params:", req.query);
+    const currentPage = "searchResults";
+    const skip =
+      parseInt(req.query.skip || "0", 10) <= 0
+        ? 0
+        : parseInt(req.query.skip, 10);
+    const limit = req.query.limit || 6;
```

Comment:
That seems great.  

```suggestion
QUESTION: since all renderings would use 6 as the value to const limit....would it make sense to move it up a block in scope so that all methods have access to it to dry the code out further...or would i make sense to keep it with the methods that use it for readability? 

    const limit = req.query.limit || 6;
```

I like moving it up in scope ATM.